README.md

```diff
@@ -45,3 +45,5 @@ example, symbols are named according to Rust conventions, enums and
 bitmasks are strongly typed, and many types have helpful `Debug`
 impls. This crate is almost entirely generated from the Khronos XML
 registry.
+
+The bindings are based off OpenXR SDK v1.0.22.
```

Comment:
I'm not sure what the value of documenting this is; it's already implicit in the submodule revision, and it's very likely to get missed in future updates.